llvm-project/stage3.cmake

```diff
@@ -15,15 +15,39 @@ set(CMAKE_C_FLAGS "--sysroot=/sysroot" CACHE STRING "")
 # Statically link resulting executable.
 set(CMAKE_EXE_LINKER_FLAGS "-static -lc++abi" CACHE STRING "")
 
+# The compiler builtins are necessary.
+set(COMPILER_RT_BUILD_BUILTINS ON CACHE BOOL "")
+
+# GWP ASAN fails to build without libexecinfo-dev. Don't need it for stage3.
+set(COMPILER_RT_BUILD_GWP_ASAN OFF CACHE BOOL "")
+
+# Don't need libfuzzer, ever.
+set(COMPILER_RT_BUILD_LIBFUZZER OFF CACHE BOOL "")
+
+# Don't need memprof, ever.
+set(COMPILER_RT_BUILD_MEMPROF OFF CACHE BOOL "")
+
+# Don't need ORC, ever.
+set(COMPILER_RT_BUILD_ORC OFF CACHE BOOL "")
+
+# Explicitly enable profiling support. The implicit default is ON.
+set(COMPILER_RT_BUILD_PROFILE ON CACHE BOOL "")
```

Comment:
hmm...I'm not sure this even built the expected library...

Reply:
Maybe related to COMPILER_RT_HAS_PROFILE.

compiler-rt/cmake/config-ix.cmake +750
compiler-rt/lib/CMakeLists.txt +53

Reply:
[186/186] Linking CXX static library lib/clang/14.0.1/lib/linux/libclang_rt.profile-x86_64.a

seems to be working, nvm

Reply:
/llvm-project/llvm/build/lib/clang/14.0.1/lib/linux/libclang_rt.profile-x86_64.a

is where it's build as part of the `compiler-rt` ninja target. `install-compiler-rt` places it in

/usr/local/lib/clang/14.0.1/lib/linux/libclang_rt.profile-x86_64.a

oh! I'm forgetting to add the install-compiler-rt to the dockerfile in this PR! d'oh!